components/Common/AnimatedPlaceholder/index.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import styles from './index.module.scss';
+
+interface Props {
+  children?: React.ReactNode;
+  width?: number;
+  height?: number;
+}
+
+const AnimatedPlaceholder = ({ children, width = 400, height = 50 }: Props) => (
+  <div
+    className={styles.placeholder}
+    style={{ width: `${width}px`, height: `${height}px` }}
+  >
+    {/* Prefer external skeleton structure or render default in case not passed */}
+    {children}
```

Comment:
```suggestion
    {children || (
      <>
        <div className={styles.placeholderImage} />
        <div className={styles.placeholderText}>
          <div className={styles.placeholderTextLine} />
          <div className={styles.placeholderTextLine} />
        </div>
      </>
    )}
```

Reply:
We can probably write it like this to prevent double checks on children

Reply:
Thanks, it looks neater! Tested as well.